.github/workflows/security-label-check.yml

```
@@ -0,0 +1,27 @@
+name: Security Label Check
+on:
+  pull_request:
+    types: [opened, edited, labeled, synchronize, unlabeled]
+jobs:
+  security-check:
+    runs-on: ubuntu-latest
+    if: 'startsWith(github.head_ref, ''contrib'') || (github.event.pull_request.head.repo.fork == true && contains(github.event.pull_request.base.ref, ''contrib''))'
```

```suggestion
    if: github.repository == 'demisto/content' && github.event.pull_request.head.repo.fork == true && contains(github.head_ref, 'xsoar-bot-contrib-ContributionTestPack') == false
```
Let's make it with the same condition as the rest of the github actions. Also, this way it is more general, in case that in the future the 'contrib' prefix will change.

---

Sure, we can do that